tests/benchmarks/meson.build

```diff
@@ -57,6 +57,12 @@ foreach t, params : tests
     path = meson.current_source_dir() / '@0@.py'.format(t)
     testname = fs.stem(path)
 
+    if params.get('suite', '').contains('ycsb') and not hse_java.found() 
```

Comment:
I would change this logic to be inline and do something like:

`'LD_LIBRARY_PATH': hse_java.found() ? fs.parent(hsejni.full_path()) : ''`